jina/serve/runtimes/gateway/grpc/__init__.py

```diff
@@ -1,57 +1 @@
-import os
-
-from jina import __default_host__
-from jina.excepts import PortAlreadyUsed
-from jina.helper import is_port_free
-from jina.serve.runtimes.gateway import GatewayRuntime
 from jina.serve.runtimes.gateway.grpc.gateway import GRPCGateway
```

Comment:
do we still need such a file ? I believe it is not a breaking change since this is internal part of the code

Reply:
__init__ always needs to be there, the question is whether we wantto make it empty and import using `from jina.serve.runtimes.gateway.grpc.gateway import GRPCGateway` or keep as it is and let users import using `from jina.serve.runtimes.gateway.grpc import GRPCGateway`